wowchemy/assets/js/wowchemy-theming.js

```diff
@@ -173,10 +173,10 @@ function renderThemeVariation(isDarkTheme, init = false) {
     if (diagramEnabled) {
       if (init) {
         /** @namespace window.mermaid **/
-        window.mermaid.initialize({theme: 'dark', securityLevel: 'loose'});
+        window.mermaid.initialize({startOnLoad:true, theme: 'dark', securityLevel: 'loose'});
       } else {
         // Have to reload to re-initialise Mermaid with the new theme and re-parse the Mermaid code blocks.
-        location.reload();
+        // location.reload();
```

Comment:
Why have you commented out this line? Line 159 also performs a similar action. The page is reloaded for a reason, so are you suggesting the new version of Mermaid works without reload in this case - i.e. that the Mermaid theme can be dynamically updated without re-initialising?

Reply:
was figuring things out and I uncommented one but forgot the other :)